app/views/admin/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1><%= @application.name %></h1>
+
+<p>Street Address: <%= @application.street_address %></p>
+
+<p>City: <%= @application.city %></p>
+
+<p>State: <%= @application.state %></p>
+
+<p>Zip Code: <%= @application.zip_code %></p>
+
+<p>Description: <%= @application.owner_description %></p>
+
+<p id="pets">Pets: 
+  <% @application.pets.each do |pet| %>
+      <br><%= link_to "#{pet.name}", "/pets/#{pet.id}" %>
+      <% if pet.find_app_relationship(@application.id).status.nil? %>
+        <%= render partial: 'button', locals: {
```

Comment:
Here's what I mean by the partial not being worth the effort for this spot: that's a lot of local vars to pass in for one little button. Is there another spot in your application where a partial could've been more powerful?